src/OpenConext/Value/Saml/Entity.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace OpenConext\Value\Saml;
+
+use OpenConext\Value\Exception\InvalidArgumentException;
+
+final class Entity
+{
+    /**
+     * @var EntityId
+     */
+    private $entityId;
+    /**
```

Comment:
Nit: (lack of) whitespace seems inconsistent with other classes of the same size.